stac_fastapi/types/stac_fastapi/types/config.py

```diff
@@ -22,6 +22,11 @@ class ApiSettings(BaseSettings):
     # `pydantic.BaseSettings` instead
     default_includes: Optional[Set[str]] = None
 
+    stac_fastapi_title: str = "stac-fastapi"
+    stac_fastapi_description: str = "stac-fastapi"
+    stac_fastapi_version: str = "0.1"
+    stac_fastapi_landing_id: str = "stac-fastapi"
```

Comment:
I've also reverted to `stac-fastapi` default as it was before 

https://github.com/stac-utils/stac-fastapi/blob/315cfaefef2ab2773ae087228439ba0371fb7372/stac_fastapi/types/stac_fastapi/types/core.py#L258

Reply:
Looks good to me. The version is the version number of the User's stac_fastapi, not this version ie. soon to be 2.5.0